packages/connect-wallet-modal/src/components/WalletsModal/components/ConnectWalletModal/sortWalletsList.ts

```diff
@@ -37,7 +42,7 @@ export async function sortWalletsList({
     if (walletsPinned.includes(walletId)) {
       // Put the pinned wallets on the first place, above all other wallets
       filteredWalletData.pinned.push(walletData);
-    } else if (await detector?.()) {
+    } else if (await detector?.(config)) {
```

Comment:
This requirement is here because of the only connector — `safe` connector. It utilizes `.getProvider()` of `connector` entity that is not accessible now without initialization wagmi config data structure. Previously, there was a possibility to call `getProvider` from ES class definition of the connector, but now there is no classes. Wagmi uses `createConnectorFn` during config description, which have no accessible methods at all. The `instance` will appear only after `createConnectorFn` initialization during `createConfig` call, and provided with `WagmiProvider` context.